t/before_all_tasks.t

```diff
@@ -0,0 +1,30 @@
+use strict;
+use warnings;
+use lib 't/lib';
+
+use Test::More;
+use Test::Deep;
+
+use Rex::Commands;
+use Rex::Commands::Run;
+
+use t::tasks::cowbefore;
```

Comment:
this caused me an error when installing, 'Finding t::tasks::cowbefore on cpanmetadb failed.'  I believe this is meant to be t::tasks::cowboy.